code/weapon/weapons.cpp

```diff
@@ -2616,6 +2616,11 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 		}
 	}
 
+	if (optional_string("$As Child Life Rand Factor:"))
```

Comment:
This is a really ugly and confusing name... but I'm not sure I have anything better. Certainly, if this applied to children of *this* weapon the naming would be much easier, but it was probably the right call to put it on the children themselves.

I would at least, spell out words like "random" and "lifetime" (also to make its like to $lifetime clearer), I think we can afford the verbosity here.

"Lifetime Random Factor When Spawned" maybe?

Reply:
> "Lifetime Random Factor When Spawned" maybe?
I'd suggest "variation factor" instead of random factor but otherwise, that seems as good as it'll get.

Reply:
Yep, that sounds viable.

I might also... Well, hmm. Is it better to have this clustered with the spawning options like it is now, or next to $lifetime?

Reply:
Here, with the spawn stuff, imo.

Reply:
Went with "Lifetime Variation Factor When Child". "Spawned" seems too ambiguous to me -- I think it's not obvious that a weapon getting created normally doesn't count as "spawning".